bundler/spec/commands/install_spec.rb

```diff
@@ -1000,6 +1000,26 @@ def run
     end
   end
 
+  context "with only option" do
+    before do
+      bundle "config set only a:b"
+    end
+
+    it "installs only gems of the specified groups" do
+      install_gemfile <<-G
+        source "#{file_uri_for(gem_repo1)}"
+        gem "rails"
```

Comment:
@r7kamura would this gem be installed with `config set only a:b`, thus would gems without any group be installed?

Reply:
`rails` gem will implicitly belong to the default group, so it will not be installed if you use `bundle config set a:b`. If you specify `bundle config set a:b:default`, the `rails` gem will also be installed.

It would have been better to explicitly include that type of expectation.

```ruby
expect(out).not_to include("Installing rails")
```